### DISCUSSION TITLE
Question about pulling drivers from session

### DISCUSSION THREAD

**Discussion by AshleyJTM** [theOehrly/Fast-F1#90] [Q&A] [answered]
Hi there, I am learning how to program python and am adopting some of your API in my chatbot. I was just wondering if there is a way I can pull data for just drivers?

This is the code I have tried however it just outputs [] with no driver details. I find it hard to understand how it to display drivers specifically and is there anyway to search for a specific driver and display information about them? 

Any help would be greatly appreciated, I love the API keep up the good work. I look forward to exploring what I can do with this and hopefully make something cool. 

```

import fastf1 as ff1

race = ff1.get_session(2017, 'Russian', 'R')
print(race.drivers)

```

**Comment by theOehrly** [comment, marked as answer]
You need to call ``race.load_laps()`` to load all (most of) the data, including the list of drivers.

It's not quite obvious, and it's one of the things that I want to improve for next season.
